receiver/sqlqueryreceiver/README.md

```diff
@@ -37,7 +37,21 @@ The configuration supports the following top-level fields:
   - [snowflake](https://github.com/snowflakedb/gosnowflake) - `username[:password]@<account_identifier>/dbname/schemaname[?param1=value&...&paramN=valueN]`
   - [sqlserver](https://github.com/microsoft/go-mssqldb) - `sqlserver://username:user_password@localhost:1433?database=db_name`
   - [tds](https://github.com/thda/tds) - `tds://username:user_password@localhost:5000/db_name`
-
+- `datasource_config` (optional): Used in place of the `datasource` string. Contains separate values that are processed separately, and then concatenated into the connection string.
```

Comment:
I don't think we need to include the information of what's going on "underneath the hood" here, I think it's fine to simply state this is an alternative connection option to the `datasource` value.